flixel/FlxGame.hx

```diff
@@ -243,6 +243,14 @@ class FlxGame extends Sprite
 		// Super high priority init stuff
 		_inputContainer = new Sprite();
 		
+		if (GameSizeX <= 0)
+		{
+			GameSizeX = Std.int(Lib.current.stage.window.width);
```

Comment:
We can't use `.window` here, needs to be `stageWidth` and `stageHeight` (Travis is currently failing because `Stage#window` doesn't exist in OpenFL legacy).